api/src/models/ImageFile.ts

```diff
@@ -71,21 +71,50 @@ class ImageFile {
     }
 
     ocrDerivative() {
-
+        let fs = require("fs");
+        let png = this.derivativePath('ocr/pngs', 'png');
+        let { exec } = require("child_process");
+        if (fs.existsSync(png)) {
```

Comment:
I think this should be `if (!fs.existsSync(png)) {` -- we want to create the file if it does not already exist.